tasktiger/tasktiger.py

```diff
@@ -691,6 +694,11 @@ def would_process_configured_queue(self, queue_name: str) -> bool:
 @click.option("-p", "--port", help="Redis server port")
 @click.option("-a", "--password", help="Redis password")
 @click.option("-n", "--db", help="Redis database number")
+@click.option(
+    "-x",
```

Comment:
`-x` can potentially be confused with `-e` as in "eXclude" a queue. I'd keep just the long option.